packages/cli/package.json

```diff
@@ -26,11 +26,14 @@
   "dependencies": {
     "@alwaysmeticulous/common": "^1.7.6",
     "@sentry/node": "^7.2.0",
+    "@types/express": "^4.17.14",
     "adm-zip": "^0.5.9",
     "archiver": "^5.3.1",
     "axios": "^0.27.2",
     "chalk": "^4.1.2",
     "cosmiconfig": "^7.0.1",
+    "express": "^4.18.1",
```

Comment:
It feels a bit wrong to make the sdk depend on `express` :thinking:

Reply:
I think depending on something to serve assets is inevitable if we want users to be able to copy and paste a command to re-run a replay locally for debugging, without having to spin up a server for the given commit. It's not too large though considering that this is a CLI, and dev connections are normally pretty fast: https://bundlephobia.com/package/express@4.18.1.